lib/perl/Genome/VariantReporting/Command/Wrappers/ModelPairFactory.pm

```diff
@@ -68,58 +68,76 @@ sub get_model_pairs {
     my %models_for_roi = %{$self->get_models_for_roi};
     while (my ($roi, $model_list) = each %models_for_roi) {
         my @models = grep {!$self->is_single_bam($_)} @{$model_list};
+        my ($discovery_build, $validation_build);
 
-        unless (@models == 2) {
-            $self->warning_message("Skipping models for ROI %s because there are not exactly two models: %s",
-                $roi, join(", ", map {$_->__display_name__} @models));
-            next;
+        if (@models == 1) {#case for no followup
+            $discovery_build = $models[0]->last_succeeded_build;
+            push @model_pairs, Genome::VariantReporting::Command::Wrappers::ModelPair->create(
+                common_translations => $self->get_common_translations(),
+                plan_file_basename => "cle_somatic_TYPE_report.yaml",
+                discovery => $discovery_build,
+                label => 'discovery',
+            );
         }
+        else {
+            unless (@models == 2) {
```

Comment:
It's a little confusing that the first thing in this `else` is another check on the same variable as in the `if`.  It might be clearer to make `@models != 2` its own `elsif` case and then leave the other logic in the `else`.